packages/babel-core/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "7.7.2",
   "description": "Babel compiler core.",
   "main": "lib/index.js",
+  "_main:used-by-babel-standalone": "src/index.js",
```

Comment:
I don't like this :laughing: 

Is it possible to use webpack's [`resolve`](https://webpack.js.org/configuration/resolve/) instead?

Reply:
By using `resolve.alias` like the other packages, webpack will ignore the `browsers` field in package.json.

Reply:
I'm not sure I follow. If we did something like this, wouldn't it work?

```js
resolve: {
  alias: {
    "@babel/core$": "packages/babel-core/src/indec.js"
  }
}
```